narwhal/primary/src/tests/state_handler_tests.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright(C) Facebook, Inc. and its affiliates.
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use super::*;
+use fastcrypto::{
+    serde_helpers::ToFromByteArray,
+    traits::{KeyPair, ToFromBytes},
+};
+use test_utils::CommitteeFixture;
+
+#[tokio::test]
+async fn start_dkg() {
+    let fixture = CommitteeFixture::builder()
+        .stake_distribution(vec![2500, 2500, 2500, 2500].into())
```

Comment:
nit - note that `nodes.reduce()` will reduce those to 50, if you want to run with with larger messages you could use different numbers

Reply:
Don't care so much about the actual values, just needed them to not be the default "1"